packages/rest/src/rest-server.ts

```diff
@@ -49,6 +55,39 @@ const OPENAPI_SPEC_MAPPING: {[key: string]: OpenApiSpecOptions} = {
   '/swagger.yaml': {version: '2.0', format: 'yaml'},
 };
 
+/**
+ * A REST API server for use with Loopback.
+ * Add this server to your application by importing the RestComponent.
+ * ```ts
+ * const app = new MyApplication({
+ *   components: [RestComponent]
+ * });
+ * // OR
+ * app.component(RestComponent);
+ * ```
+ *
+ * To add additional instances of RestServer to your application, use the
+ * `.server` function:
+ * ```ts
+ * app.server(RestServer, 'nameOfYourServer');
+ * ```
+ *
+ * By default, one instance of RestServer will be created when the RestComponent
+ * is bootstrapped. This instance can be retrieved with
+ * `app.getServer(RestServer)`, or by calling `app.get('servers.RestServer')`
+ * Note that retrieving other instances of RestServer must be done using the
+ * server's name:
+ * ```ts
+ * let server = await app.getServer(RestServer, 'foo')
```

Comment:
I think this example is not correct, `getServer`  takes a single argument only -
 see https://github.com/strongloop/loopback-next/blob/02e2ce0a031367438afeaeabdfae2e29e10f25ee/packages/core/src/application.ts#L113-L115

Did you perhaps mean this?

```ts
const server = await app.getServer<RestServer>('foo');
```

Similarly, `app.get('servers.RestServer')` above should be probably `app.get<RestServer>('servers.RestServer')`.

Also we should be using `const` not `let` in general - see http://loopback.io/doc/en/contrib/style-guide.html#variable-declarations

Reply:
I used `let` in this example because I'm technically reusing the variable reference from above. It would not compile if it were a `const`.

Reply:
It won't compile with `let` either.

```
$ node
> let a = 1
undefined
> let a = 2
SyntaxError: Identifier 'a' has already been declared
```